service-tests/github.js

```diff
@@ -132,6 +132,70 @@ t.create('Tag')
     value: Joi.string()
   }));
 
+t.create('Package version')
+  .get('/package-json/v/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('package'),
+    value: Joi.string().regex(/^v\d+(\.\d+)?(\.\d+)?$/)
+  }));
+
+t.create('Package name')
+  .get('/package-json/n/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('package name'),
+    value: Joi.equal('gh-badges')
+  }));
+
+t.create('Package name - Custom label')
+  .get('/package-json/name/badges/shields.json?label=Dev Name')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('Dev Name'),
+    value: Joi.equal('gh-badges')
+  }));
+
+t.create('Package object/invalid')
+  .get('/package-json/keywords/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('package keywords'),
+    value: Joi.equal('invalid data')
```

Comment:
For an array that is one level deep, how about joining the values with `', '` and displaying them?

Reply:
Will have a look at this soon and modify it.
Should i make the output the same as #820?
basically just stringify the value

Reply:
I think it's better to stringify the elements but make the list more like what a human would like to see: **1, 2, 3** is better than **[1, 2, 3]**. If something is deeply nested that is probably a mistake. We could detect it and report an error, but I don't think it's that important.

Reply:
Are you up for looking at this? ^^

Reply:
Yup should be able to create a PR later tonight/tomoro when I'm off mobile.
Thinking just something like

If (typeof value == "object")
  value = value.join(", ");

Which should just err out and return "invalid data" if it's not an array?

Reply:
Awesome, thanks. You could use [`Array.isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray).